index.php

```diff
@@ -0,0 +1,30 @@
+<?php include('connect.php'); ?><!DOCTYPE html>
+<html>
+  <head>
+    <title>Make Your Pizza</title>
+  </head>
+  <body>
+    <div class="container">
+      <h1>Choose Your Toppings</h1>
+      <form action="review.php" method="POST">
+        <div class="group">
+          <label for="cheese">Cheese</label><?php while($row = mysqli_fetch_assoc($results['cheeses'])) { ?>
+          <input name="cheese" type="radio" value="&lt;?php echo $row['name']; ?&gt;"><span><?php echo $row['name'] ?>: $<?php echo $row['price']; ?></span><?php } ?>
+        </div>
```

Comment:
I think you should change this:
```php
<input name="cheese" type="radio" value="&lt;?php echo $row['name']; ?&gt;">
```
to this:
```php
<input name="cheese" type="radio" value="<?php echo $row['name']; ?>">
```

I'm not sure, but if you run your code, open your browser and inspect element it probably looks like: `value="&lt;?php echo $row['name']; ?&gt;"` rather than `value="Mozzarella"`.  PHP is processed before it touches the browser and it doesn't pay attention to code outside of <?php ?> so it won't know it's inside quotes in the value attribute.